packit/config/package_config.py

```diff
@@ -230,6 +235,23 @@ def get_deprecated_key(raw_dict: dict, new_key_name: str, old_key_name: str):
             r = old
         return r
 
+    def get_all_files_to_sync(self):
+        """
+        Adds the default files (config file, spec file) to synced files when doing propose-update.
+        :return: SyncFilesConfig with default files
+        """
+        files = self.synced_files.files_to_sync
+
+        if self.specfile_path not in (item[0] for item in files):
+            files.append(SyncFilesItem(src=self.specfile_path, dest=self.specfile_path))
+
+        if self.config_file_path not in (item[0] for item in files):
```

Comment:
`item.src` would be a bit better than `item[0]`
(That's why `SyncFilesItem` inherits from `NamedTuple`)